openmetadata-service/src/main/java/org/openmetadata/service/JsonPatchProvider.java

```diff
@@ -26,6 +27,9 @@ public ObjectMapper getContext(Class<?> type) {
     final ObjectMapper mapper = new ObjectMapper();
     mapper.registerModule(new JSR353Module());
     mapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
+    // This feature allows the parser to accept non-numeric numbers such as NaN,
+    // Infinity, and -Infinity in JSON input.
```

Comment:
can you add a link to this wiki https://github.com/FasterXML/jackson-core/wiki/JsonReadFeatures please?